composer.json

```
@@ -15,7 +16,13 @@
         }
     ],
     "require": {
-        "php": "^8.0"
+        "php": "^8.0",
+        "spatie/ignition": "^1.0",
+        "symfony/dependency-injection": "^4.4 || ^5.4 || ^6.0",
+        "symfony/config": "^4.4 || ^5.4 || ^6.0",
+        "symfony/http-kernel": "^4.4 || ^5.4 || ^6.0",
+        "symfony/event-dispatcher": "^4.4 || ^5.4 || ^6.0",
+        "symfony/http-foundation": "^4.4 || ^5.4 || ^6.0"
```

As we require `spatie/ignition` Symfony `^4.4` is not installable anyway as they only support Symfony `^5.4 || ^6.0`.
```suggestion
        "symfony/dependency-injection": "^5.4 || ^6.0",
        "symfony/config": "^5.4 || ^6.0",
        "symfony/http-kernel": "^5.4 || ^6.0",
        "symfony/event-dispatcher": "^5.4 || ^6.0",
        "symfony/http-foundation": "^5.4 || ^6.0"
```